party/implementation.go

```diff
@@ -352,11 +415,11 @@ func (p *Impl) computeCommittee(trackid *common.TrackingID) (common.SortedPartyI
 }
 
 func (p *Impl) committeeSize() int {
```

Comment:
Should CommitteeSize to be protocol aware (ECDSA or EDDSA)?
If yes then we should have that passed it/have two functions. Right now it's weird since if there is no frostConfig, for example, then it requires n-out-of-n.